tests/integration/dynamic_ip_test.py

```diff
@@ -1501,6 +1501,7 @@ def clean_state():
         if iface_state.get(Interface.IPV4, {}).get(InterfaceIPv4.ENABLED):
             iface_state[Interface.IPV4][InterfaceIPv4.AUTO_DNS] = False
             iface_state[Interface.IPV4][InterfaceIPv4.AUTO_ROUTES] = False
+            iface_state[Interface.IPV4].pop(InterfaceIPv4.FORWARDING, None)
```

Comment:
It is very important for nmstate to support applying state produced by `nmstatectl show`.

There are two options:
A: Only show `forwarding` in query when NM support so.
B: NM discard `forwarding` settings when desired but not support, let verification stage found the error.

I prefer option B, with that, you don't need to change existing test codes.